### DISCUSSION TITLE
Problème de position d'un acteur par rapport à une image

### DISCUSSION THREAD

**Discussion by Mathis554** [witsa/synapps#744] [Q&A] [answered]
Bonjours,
J'ai une image d'un plant de bâtiment
![Bat 12 RDC](https://github.com/witsa/synapps/assets/174346872/870091bc-856e-4945-ae9e-40c2b49cbd58)
Par-dessus j'ai mis un acteur qui indique quand la lumière est allumée
J'ai positionné l'acteur dans une pièce à l'aide de l'acteur canva
![image](https://github.com/witsa/synapps/assets/174346872/889c88fe-7678-47c6-912e-4401933be3fa)
![image](https://github.com/witsa/synapps/assets/174346872/a8d856fd-c71a-4261-91db-fd36d9db907e)
Le problème est que quand je mets le synapps sur le navigateur l'image se redimensionne à la taille de l'écran est l'acteur lumière n'est plus dans la pièce
![image](https://github.com/witsa/synapps/assets/174346872/cfec8bf2-6ffc-4e89-89ae-d15dd75b7607)
Je ne sais pas comment faire pour que la lumière reste dans la pièce quelle que soit la dimension de l'écran
Merci d'avance

**Comment by Raphwit** [comment, marked as answer]
Bonjour, il vous suffit d'utiliser un acteur "Boite à vue", il faut que votre acteur "Canvas" se situe à l'intérieur de cette boite à vue pour que ce dernier soit redimensionné automatiquement.

Voici les conseils d'usages pour cet acteur :
https://witsa.github.io/synapps/concepts/actor-types/layout-view-box.html#usage 

Attention, n'oubliez pas de définir une taille pour l'acteur Canvas !

**Comment by Mathis554** [comment]
Merci, beaucoup la fonctionne aux petits oignons
